docs/contributing.md

```diff
@@ -293,7 +293,7 @@ if you are adding a new page, or doing a major rewrite.
 
 ### Install `uv`
 
-The only thing you need is [`uv`][uv] (version 0.5.1 or newer).
+The only thing you need is [`uv`][uv] (version 0.7.3 or newer).
```

Comment:
I don't know that there is much benefit to updating the minimal version of `uv` at this point. Does it buy us anything? Note that another CI job checks that everything works with the latest version of `uv`.

Reply:
Reverted